api/v3/Mailing.php

```diff
@@ -637,7 +637,7 @@ function civicrm_api3_mailing_send_test($params) {
     $query = CRM_Utils_SQL_Select::from('civicrm_email e')
         ->select(array('e.id', 'e.contact_id', 'e.email'))
         ->join('c', 'INNER JOIN civicrm_contact c ON e.contact_id = c.id')
-        ->where('e.email IN (@emails)', array('@emails' => $testEmailParams['emails']))
+        ->where('LOWER(e.email) IN (@emails)', array('@emails' => $testEmailParams['emails']))
```

Comment:
actually we don't need this LOWER - mysql is case insensitive - it's php we need to wrangle.

Note that using LOWER will significantly slow down the query - a slightly similar example is in this blog https://civicrm.org/blog/eileen/guppies-have-died-for-that-query (diff function but same underlying issue)